tests/integration/devfile/cmd_dev_test.go

```diff
@@ -54,7 +54,7 @@ var _ = Describe("odo dev command tests", func() {
 			Expect(helper.VerifyFileExists(".odo/env/env.yaml")).To(BeFalse())
 		})
 		It("should show validation errors if the devfile is incorrect", func() {
-			err := helper.RunDevMode("", func(session *gexec.Session, outContents, errContents []byte, matches []string) {
+			err := helper.RunDevMode(func(session *gexec.Session, outContents, errContents []byte, matches []string) {
```

Comment:
You could replace `matches` with `urls` to be more explicit

Reply:
I am clearly in a rush to get this PR in. Or may I say, in a rush to get rid of this PR. 😄

Reply:
Done d4ee0f5.